scripts/sct_download_data.py

```diff
@@ -38,6 +29,7 @@ def get_parser():
         type_value="multiple_choice",
         description="Name of the dataset.",
         mandatory=True,
+        # TODO: replace with key dict items
```

Comment:
Are you thinking of loading the docstring directly from `DICT_URL`? `example=sorted(DICT_URL)`?

Reply:
yup! fixed in 9d531465638a9b7299f52526eb7b2ba36b0cc5b3